src/controllers/orderController.js

```diff
@@ -0,0 +1,42 @@
+const { Order, OrderItem } = require('../models/Order');
```

Comment:
Per the Github Action annotation; we can remove `OrderItem` here since it's not used and is not exported by Order.js, and just have `const { Order } = require('../models/Order');`

Reply:
Removed